makefile

```diff
@@ -31,7 +30,8 @@ include make/libraries
 # Set default compiler options.
 ##
 CFLAGS = -I . -isystem $(EIGEN) -isystem $(BOOST) -isystem$(CVODES)/include -Wall -DBOOST_RESULT_OF_USE_TR1 -DBOOST_NO_DECLTYPE -DBOOST_DISABLE_ASSERTS -DNO_FPRINTF_OUTPUT -pipe
-CFLAGS_GTEST = -DGTEST_USE_OWN_TR1_TUPLE
+CXXFLAGS = -std=c++14
```

Comment:
`CXXFLAGS` is _just_ for the C++ compiler while `CFLAGS` is just for the C compiler. This only matters because cvodes (and maybe other libraries?) are not C++. Cvodes is actually Objective C, apparently. And `CFLAGS` are sent to its compiler (creating errors) but `CXXFLAGS` are not.